src/getHumanAge.test.js

```diff
@@ -7,4 +7,29 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 when catsAge = 0 and dogsAge = 0`, () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test(`should return 0 for catsAge < 15 and dogsAge < 15`, () => {
```

Comment:
Change the variable names 'catsAge' and 'dogsAge' to 'catAge' and 'dogAge' in the test description to match the task description and function parameters.